plugins/modules/foreman_auth_source_ldap.py

```diff
@@ -209,7 +209,7 @@ def main():
 
     module.connect()
 
-    entity = module.find_resource_by_name('auth_source_ldaps', name=entity_dict['name'], failsafe=True)
+    entity = module.find_resource_by_name('auth_source_ldaps', name=entity_dict['name'], failsafe=True, thin=False)
```

Comment:
Shouldn't we rely on the default "magic" for `thin` here?

Reply:
Only if we want to rerecord that here. And i'd like to postpone that to when refactoring all those modules that slipped in in the meantime.

Reply:
fair, could you add a comment to the code stating so? otherwise we'll wonder the same next time we touch the module.